civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -569,9 +569,40 @@ function civihr_employee_portal_init() {
     _rebuild_view('appraisal');
     _rebuild_view('tasks');
     _rebuild_view('documents');
+
+    _load_ta_settings();
+
+    // Page load scripts
+    civicrm_resources_load('org.civicrm.reqangular', ['reqangular.min.js']);
+    _add_script_to_page('/js/ta-documents-app.js', 'dashboard');
```

Comment:
please move these two lines adding scripts closer to the existing `_add_script_to_page` calls, before the `_rebuild_view` calls.

The 'Page load scripts' comment isn't adding any useful information. Please remove it.